diff --git a/node_modules/@graphql-tools/merge/esm/typedefs-mergers/merge-nodes.js b/node_modules/@graphql-tools/merge/esm/typedefs-mergers/merge-nodes.js
index 1203c26..5b0e530 100644
--- a/node_modules/@graphql-tools/merge/esm/typedefs-mergers/merge-nodes.js
+++ b/node_modules/@graphql-tools/merge/esm/typedefs-mergers/merge-nodes.js
@@ -1,5 +1,4 @@
 import { Kind, } from 'graphql';
-import { isNode } from 'graphql/language/ast.js';
 import { collectComment } from '@graphql-tools/utils';
 import { mergeDirective } from './directives.js';
 import { mergeEnum } from './enum.js';
@@ -13,6 +12,8 @@ export const schemaDefSymbol = 'SCHEMA_DEF_SYMBOL';
 export function isNamedDefinitionNode(definitionNode) {
     return 'name' in definitionNode;
 }
+import pkg from 'graphql/language/ast.js';
+const { isNode } = pkg;
 export function mergeGraphQLNodes(nodes, config, directives = {}) {
     const mergedResultMap = directives;
     for (const nodeDefinition of nodes) {
